pandas/io/formats/printing.py

```diff
@@ -283,9 +284,14 @@ def format_object_summary(obj, formatter, is_justify=True, name=None):
         string formatter for an element
     is_justify : boolean
         should justify the display
-    name : name, optiona
+    name : name, optional
         defaults to the class name of the obj
 
+        Pass ``False`` to indicate that subsequent lines should
+        not be indented to align with the name.
+    trailing_comma : bool, default True
```

Comment:
see if we can have a more meaningful name here, e.g. this is a multi-line repr yes?

Reply:
Not quite... since we could have a one-liner with or without a trailing comma

```
In [6]: pidx
Out[6]: PeriodIndex(['2000-01-01'], dtype='period[D]', freq='D')
```

So that has a trailing comma after values, since it's inside the `PeriodIndex` constructor. But 


```python
In [8]: pidx.array
Out[8]:
<PeriodArray>
['2000-01-01']
Length: 1, dtype: period[D]
```

doesn't have one, since it's not inside the constructor. I don't know if the hypothetical EA-one-line repr needs one or not.

```
# <PeriodArray>['2000-01-01'] dtype: period[D]
# <PeriodArray>['2000-01-01'], dtype: period[D]
```

Reply:
If you prefer, I can remove `trailing_comma` entirely, and just `.rstrip(', \n')` in the caller that doesn't need it.

Reply:
yeah maybe that's better